<?xml version="1.0" ?>
<launch>
  <arg name="subjectName" default="walid"/>
	<arg name="executionMode" default="c"/>
	<arg name="trackingMode" default="t"/>
	<arg name="fittingMethod" default="p"/>
	<arg name="useKinect" default="false"/>
  
  <group unless="$(arg useKinect)">
    <node output = "screen" pkg="hp_preliminary_experiment" type="preliminaryExperiment" name="preliminaryExperiment" args="$(arg subjectName) -e $(arg executionMode) -t $(arg trackingMode) -f $(arg fittingMethod)"/>
  </group>

  <group if="$(arg useKinect)">
    <node output = "screen" pkg="hp_preliminary_experiment" type="kinectRecording" name="kinectRecording" args="$(arg subjectName) -e $(arg executionMode) -f $(arg fittingMethod)"/>
  </group>


</launch>
